requirements.txt

```diff
@@ -1,7 +0,0 @@
-flask==1.0.2
```

Comment:
You mustn't delete this file: this is used by Heroku to install all dependencies